SPARQLWrapper/Wrapper.py

```diff
@@ -507,7 +507,7 @@ def _createRequest(self):
         if self.user and self.passwd:
             if self.http_auth == BASIC:
                 credentials = "%s:%s" % (self.user, self.passwd)
-                request.add_header("Authorization", "Basic %s" % base64.b64encode(credentials.encode('utf-8')))
+                request.add_header("Authorization", "Basic %s" % base64.b64encode(credentials.encode('utf-8')).decode('utf-8'))
```

Comment:
and having said that, didn't we just have an issue where adding headers as unicode broke quite a lot of things? maybe the underlying problem here is that all the headers we add should be byte strings...